view/theme/frio/js/theme.js

```diff
@@ -764,12 +764,52 @@ function htmlToText(htmlString) {
  * @param {boolean} un    Whether to perform an activity removal instead of creation
  */
 function doActivityItemAction(ident, verb, un) {
-	if (verb.indexOf("attend") === 0) {
-		$(".item-" + ident + " .button-event:not(#" + verb + "-" + ident + ")").removeClass("active");
-	}
-	$("#" + verb + "-" + ident).toggleClass("active");
+  console.log(ident, verb, un);
+	_verb = un ? 'un' + verb : verb;
+	$('#like-rotator-' + ident.toString()).show();
+	$.post('item/' + ident.toString() + '/activity/' + _verb)
+	.success(
+		function(data){
+      console.log("data.status: " + data.status);
+      if (data.status == "ok") {
+        console.log("connection: " + data.status);
+        $('#like-rotator-' + ident.toString()).hide();
+        if (verb.indexOf("announce") === 0 ) {
+          console.log("announce")
+          if (data.verb == "un" + verb) {
+            $("button[id^=shareMenuOptions-" + ident.toString() + "]" ).removeClass("active");
+            $("button[id^=shareMenuOptions-" + ident.toString() + "]" ).attr("onclick", "doActivityItemAction(" + ident +", '" + verb + "', " + false + ")").change();
+            $("button[id^=" + verb + "-" + ident.toString() + "]" ).removeClass("active");
+            $("button[id^=" + verb + "-" + ident.toString() + "]" ).attr("onclick", "doActivityItemAction(" + ident +", '" + verb + "', " + false + ")").change();
```

Comment:
In this case we want the boolean string because it's used as a parameter value. Please fix this in the other cases.
```suggestion
            $("button[id^=" + verb + "-" + ident.toString() + "]" ).attr("onclick", "doActivityItemAction(" + ident +", '" + verb + "', false)").change();
```

Reply:
prove me wrong... 

But i use it as boolean. This was my intention.
In your suggestion the quoting is wrong.

Reply:
Just look at how it is written in the `onclick` attribute of the current HTML of the action button:
```html
<button type="button" class="btn-link button-likes active" aria-pressed="true" id="like-5705380" title="I like this (toggle)" onclick="doActivityItemAction(5705380, 'like', true);" data-toggle="button"><i class="fa fa-thumbs-up" aria-hidden="true"></i>&nbsp;Like</button>
```
The boolean parameter value is spelled out `true` or `false`. Since you are also setting the `onclick` attribute value to a string in your code, this parameter value needs to be a string.

And my quoting is correct, I believe you may be confused by the nearby single quote for the `verb` parameter.